CHANGELOG.md

```diff
@@ -352,7 +352,7 @@
 * Support legacy devices without a Name property
 * (Covered) Crash in agent's passkey / pin code methods
 * Fix fading in GtkAnimation TreeRow(Color)Fade and CellFade classes.
-* Do not explixitely destoy DeviceSelectorDialog blueman-sendto.
+* Do not explixitely destroy DeviceSelectorDialog blueman-sendto.
```

Comment:
Well... 😄

Reply:
Weird it got the destoy but not `explixitely ` -> `explicitly`. @kianmeng can you add this change?

Reply:
@infirit FYI, https://github.com/codespell-project/codespell/pull/2691

Reply:
Nice 👍